subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1103,6 +1103,21 @@ int ull_conn_llcp(struct ll_conn *conn, uint32_t ticks_at_expire, uint16_t lazy)
 			conn->llcp_type = LLCP_CONN_UPD;
 			conn->llcp_ack -= 2U;
 
+#if defined(CONFIG_BT_CTLR_PERIPHERAL_ISO)
+		} else if (conn->llcp_cis.req != conn->llcp_cis.ack) {
+			if (conn->llcp_cis.state == LLCP_CIS_STATE_RSP_WAIT) {
+				struct lll_conn *lll = &conn->lll;
```

Comment:
```suggestion
				const struct lll_conn *lll = &conn->lll;
```

Reply:
Const is better, yes. Will change. However, const pattern is not used for construction this elsewhere in ull_conn.c.